### ISSUE TITLE
Using target dependency condition results in parsing error

### ISSUE THREAD

**Issue by jonasrottmann** [simonbs/dependency-graph#22]
### What happened?

Using a [Target Dependency Condition](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#target-dependency-condition) results in an error `Error: Failed parsing dumped Package.swift file at ...`.

### What are the steps to reproduce?

1. Update a target dependency to use a condition, for example:
```swift
...
dependencies: [
    .product(
    	name: "Name", 
    	package: "package", 
    	condition: .when(platforms: [.iOS])
    ),
    ...
]
...
```

2. Run [dependency-graph](https://github.com/simonbs/dependency-graph)

### What is the expected behavior?

The `Package.swift` file should be parsed without an error. The condition of the dependency does probably not need to be evident from the resulting visualisation.

**Comment by simonbs** [comment]
@jonasrottmann Can this be closed now that #23 is merged?

**Comment by jonasrottmann** [comment]
Yes 👏 Thank you!
